### PR TITLE
docs(argocd-image-updater): adding google artifact registry to the list

### PR DESCRIPTION
**Pull request by Kripu77**
Checklist:

* [x] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#versioning)
* [x] I have updated the documentation according to [documentation](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#documentation)
* [x] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#changelog).
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/blob/master/community/CONTRIBUTING.md).
* [x] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/stable/developer-guide/ci/)).

### CODE REVIEW

**Review by yu-croco — CHANGES_REQUESTED** [review]
Please handle below (items on your PR description)
> I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#versioning)

> I have updated the [documentation](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#documentation) according to documentation

-> items are supposed to be written in README.md.gotmpl and run script.

> I have updated the chart [changelog](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#changelog) with all the changes that come with this pull request according to changelog.

**Comment by Kripu77** [comment]
Hi [jmeridth](https://github.com/jmeridth), Apologies for the delay. I have updated the PR as requested.

Thank you!

File: charts/argocd-image-updater/Chart.yaml [hunk 1]
```diff
@@ -2,21 +2,25 @@ apiVersion: v2
...
-  - argocd
-  - gitops
+- argoproj
+- argocd
+- gitops
 maintainers:
-  - name: argoproj
-    url: https://argoproj.github.io/
+- name: argoproj
+  url: https://argoproj.github.io/
```
Comment (yu-croco) [inline charts/argocd-image-updater/Chart.yaml:15]: It looks we don't need these diffs 👀

File: charts/argocd-image-updater/Chart.yaml [hunk 2]
```diff
@@ -2,21 +2,25 @@ apiVersion: v2
...
   artifacthub.io/signKey: |
     fingerprint: 2B8F22F57260EFA67BE1C5824B11F800CD9D2252
     url: https://argoproj.github.io/argo-helm/pgp_keys.asc
   artifacthub.io/changes: |
     - kind: added
       description: Introduce chart signing
+    - kind: added
+      description: Added Google Artifact Repository in the list of supported registries.
+    - kind: fixed
+      description: Fixed a typo in SECURITY.md
```
Comment (yu-croco) [inline charts/argocd-image-updater/Chart.yaml:26]: >Fixed a typo in SECURITY.md   

It's not about argocd-image-updater, so I wonder this change is included in the same PR. 🤔

Comment (Kripu77) [reply charts/argocd-image-updater/Chart.yaml]: Apologies, will raise another one for the typo fix.

**Review by yu-croco — APPROVED** [review]
LGTM

**Comment by yu-croco** [comment]
Hi @Kripu77 , please resolve conflict when you have time :)

File: SECURITY.md [hunk 3]
```diff
@@ -2,8 +2,10 @@
 
 ## Supported Versions and Upstream Reporting
 
+
 Each helm chart currently supports the designated application version in the Chart.yaml. There is a chance a security issue you've discovered may not be with the helm chart but with the upstream application. Please visit that application's Security policy document to find out how to report the security issue.
 
+
```
Comment (yu-croco) [inline SECURITY.md:8]: nits;
maybe unnecessary diffs 👀

**Pull request closed without merging**